rust/arrow/src/builder.rs

```diff
@@ -519,6 +519,16 @@ impl BinaryBuilder {
         Ok(())
     }
 
+    /// Appends a byte slice into the builder.
+    ///
+    /// Automatically calls the `append` method to delimit the slice appended in as a
+    /// distinct array element.
+    pub fn append_values(&mut self, value: &[u8]) -> Result<()> {
```

Comment:
It seems the `BinaryBuilder` is poorly named - the original `append_value` should be called something else since it only append a single byte to the binary array. This instead should better be called `append_value`.

Reply:
I agree on this, and it gets a bit confusing because I would think that appending a "value" would be more common than appending bytes one-by-one. How about the more direct `append_byte`?

Reply:
Sure, `append_byte` sounds better.